composer.json

```diff
@@ -28,6 +28,7 @@
         "bamarni/composer-bin-plugin": "^1.8",
         "dama/doctrine-test-bundle": "^7.0|^8.0",
         "doctrine/common": "^3.2",
+        "doctrine/collections": "^2.0",
```

Comment:
I added this because of this error: https://github.com/zenstruck/foundry/actions/runs/8572205459/job/23494200535

I don't know why it occurs now.... but even with php 8.1 it happen...
it's just cheating with tests permutations :innocent: